pkg/controller/proxyconfig/controller.go

```diff
@@ -232,7 +232,8 @@ func (r *ReconcileProxyConfig) Reconcile(request reconcile.Request) (reconcile.R
 		}
 
 		// Create a configmap containing the merged proxy.trustedCA/system bundles.
-		trustBundle, err = r.mergeTrustBundlesToConfigMap(proxyData, systemData)
+		_, err = r.mergeTrustBundlesToConfigMap(proxyData, systemData)
```

Comment:
why do you think this was a mistake? (also fwiw the variable name is wrong in the commit message: "trustedBundle" rather than "trustBundle")

Reply:
i thought it might be a mistake because the value of `trustBundle` is never used.
then again, `mergeTrustBundlesToConfigMap` could have some side effect that's important and the return value isn't needed.

Reply:
Hm... I was thinking it got used [after the `switch`](https://github.com/openshift/cluster-network-operator/blob/27e5e510/pkg/controller/proxyconfig/controller.go#L319), but reading more carefully, it seems that it's guaranteed to be overwritten before that point (either [here](https://github.com/openshift/cluster-network-operator/blob/27e5e510/pkg/controller/proxyconfig/controller.go#L279) or [here](https://github.com/openshift/cluster-network-operator/blob/27e5e510/pkg/controller/proxyconfig/controller.go#L300) depending on `!isSpecTrustedCASet(&proxyConfig.Spec)`)

This may need further investigation. (Also, @alexanderConstantinescu this seems maybe kinda related to #533?)

Reply:
Could be, I don't know enough about this proxy stuff to stick my hand in the fire with confidence. However, this reconciliation loop finishes with `r.syncTrustedCABundle(trustBundle)` which just updates the `trusted-ca-bundle` configmap with the new data. This is de-corrallated from #533 where the point of that PR is to update _every other_ config map in the cluster with CA data, which is done `configmap_ca_injector/controller.go`

Reply:
/cc @danehans